pinot-tools/src/main/resources/log4j2.xml

```diff
@@ -28,7 +28,7 @@
       https://logging.apache.org/log4j/2.x/manual/pattern-layout.html#converter-not-empty
       and https://logging.apache.org/log4j/2.x/manual/pattern-layout.html#converter-thread-context-map
     -->
-    <Property name="LOG_PATTERN">%d{yyyy/MM/dd HH:mm:ss.SSS} %p [%c{1}] [%t]%notEmpty{ [cid=%X{pinot.query.cid}%notEmpty{,stg=%X{pinot.mse.stageId}}]} %m%n</Property>
+    <Property name="LOG_PATTERN">%d{yyyy/MM/dd HH:mm:ss.SSS} %p [%c{1}] [%t]%notEmpty{ [cid=%X{pinot.query.cid}%notEmpty{,queryHash=%X{pinot.query.hash}}%notEmpty{,stg=%X{pinot.mse.stageId}}]} %m%n</Property>
```

Comment:
Same here, can we use a shorter literal than `queryHash`? Maybe `fgp`, just `hash` or `shape`?

Also, this is not a hash of the query but the hash of the fingerprint

Reply:
updated it to hash